game/src/plugins/skills/mining/src/gem.kt

```diff
@@ -2,9 +2,12 @@ package org.apollo.game.plugin.skills.mining
 
 data class Gem(val id: Int, val chance: Int)
```

Comment:
We should use enums for gems/pickaxes

Reply:
Pickaxes and Gems can be enums. But I feel it over complicates things a little.